cpg_workflows/workflow.py

```diff
@@ -682,7 +667,6 @@ def _get_action(self, target: TargetT) -> Action:
                 logging.info(f'{self.name}: {target} [REUSE] (expected outputs exist: {expected_out})')
                 return Action.REUSE
 
-        logging.info(f'{self.name}: {target} [QUEUE]')
```

Comment:
This is repeated on line 1324 in `queue_for_multicohort` - we log the name, multicohort, and action.name for every stage in the analysis. The result is that this line is printed twice for every Stage instance

Reply:
Funnily, we made this call in a separate PR around the same time.